Formula/c/cdi.rb

```diff
@@ -0,0 +1,49 @@
+class Cdi < Formula
+  desc "C and Fortran Interface to access Climate and NWP model Data"
+  homepage "https://code.mpimet.mpg.de/projects/cdi"
+  url "https://code.mpimet.mpg.de/attachments/download/28877/cdi-2.2.4.tar.gz"
+  sha256 "bfa632fe27e04a84d743a6a4d2036488edf725a756d5688058704a9e18da2411"
+  license "GPL-2.0-only"
+
+  livecheck do
+    url "https://code.mpimet.mpg.de/projects/cdi/files"
+    regex(/href=.*?cdi[._-]v?(\d+(?:\.\d+)+)\.t/i)
+  end
+
+  depends_on "eccodes"
+  depends_on "hdf5"
+  depends_on "libaec"
+  depends_on "netcdf"
+  depends_on "proj"
+  uses_from_macos "python" => :build
+
+  def install
+    args = %W[
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --prefix=#{prefix}
```

Comment:
```suggestion
```

Reply:
`--disable-silent-rules` is not part of `std_configure_args`, but the others is yes